scanner/README.md

```diff
@@ -0,0 +1,85 @@
+# Scanner Grafana Dashboards
+
+There are four dashboards to monitor scanning information
+
+- `Appscode / Scanner / Cluster CVEs`: Shows vulnerability info in cluster level.
+- `Appscode / Scanner / Namespace CVEs`: Shows vulnerability info in namespace level.
+- `Appscode / Scanner / Image CVEs`: Shows vulnerability info for a particular image.
+- `Appscode / Scanner / App CVEs`: Shows vulnerability info for a particular workload. Ex: MongoDB, Deployment, Replicaset etc.
+
+
+Note: These dashboards are developed in **Grafana version 9.3.1**
```

Comment:
I wonder, grafana versioned 9.x.x dashboard will perfectly work in 7.x.x or 8.x.x grafana version?

Reply:
@ArnobKumarSaha can you please check these dashboard json is also working in grafana 7.x.x and 8.x.x ?